examples/apache/matrix-synapse.conf

```diff
@@ -56,10 +56,6 @@
 	<Files "/matrix/static-files/.well-known/matrix/server">
 		Require all granted
 	</Files>
-	<Location "/.well-known/matrix/server>
```

Comment:
Perhaps we still need the `Header always set Content-Type "application/json" header?

The [Server-Server spec](https://matrix.org/docs/spec/server_server/r0.1.3#get-well-known-matrix-server) says:

> The Content-Type for this response SHOULD be application/json, however servers parsing the response should assume that the body is JSON regardless of type.

.. so it doesn't seem to be strictly necessary to serve it with `Content-Type: application/json`, but it's better if it is.